examples/wind-polar-selections.py

```diff
@@ -0,0 +1,96 @@
+from dash import Dash, html, dcc, callback, Input, Output
+import dash_bootstrap_components as dbc
+import plotly.express as px
+
+app = Dash(__name__, external_stylesheets=[dbc.themes.CYBORG])
+
+df = px.data.wind()
+df[["min_strength", "max_strength"]] = (
+    df.strength.str.replace(r"6\+", "6-7").str.split("-", expand=True).astype(int)
+)
+
+dropdown_1 = dbc.Select(
+    options=[{"label": v, "value": v.lower()} for v in ("Bar", "Line")],
+    value="bar",
+    required=True,
+)
+
+range_1 = dcc.RangeSlider(
+    0,
+    7,
+    step=1,
+    marks={**{v: str(v) for v in range(7)}, 7: ">6"},
```

Comment:
Should also add a value:
`value=[0, 7]`

Reply:
Done.